docs/docs/general/interactions/events.md

```diff
@@ -22,3 +22,23 @@ var chart = new Chart(ctx, {
     }
 });
 ```
+
+## Converting Events to Data Values
+
+A common occurrence is taking an event, such as a click, and finding the data coordinates on the chart where the event occurred. Chart.js provides helpers that make this a straightforward process.
+
+```javascript
+const chart = new Chart(ctx, {
+    type: 'line',
+    data: data,
+    options: {
+        onClick: (e) => {
+            const canvasPosition = Chart.helpers.getRelativePosition(e);
```

Comment:
I'm having a problem following this example, using 2.9.4 (in FF 84 if that matters):
```
Uncaught TypeError: e is undefined
    getRelativePosition .../Chart.js_2.9.4.min.js:7
    onClick .../:315
Chart.js_2.9.4.min.js:7:169582
```
(Translates to
```
Uncaught TypeError: chart is undefined
    getRelativePosition .../Chart.js_2.9.4.js:10677
    onClick .../:315
```
not minified.)

I'm not fluent in JS, so it might be me. But when I pass the second argument (chart) to https://github.com/chartjs/Chart.js/blob/2efffb8ae4b86045100252186cadaf41a1b1d93e/src/helpers/helpers.dom.js#L74 it works much better.

Reply:
Good catch. I updated the docs in https://github.com/chartjs/Chart.js/pull/8336

Reply:
So chart arg is expected - thank you for confirmation and docs update.